pkg/filters/binary.go

```diff
@@ -39,15 +36,9 @@ func getHostMntNS() (uint32, error) {
 	var ns int
 	var err error
 
-	err = capabilities.GetInstance().Requested(func() error {
-		ns, err = proc.GetProcNS(1, "mnt")
-		return errfmt.WrapError(err)
-	},
-		cap.DAC_READ_SEARCH,
-		cap.SYS_PTRACE,
-	)
+	ns, err = proc.GetProcNS(1, "mnt")
```

Comment:
Doesn't this function require capabilities?

Reply:
Execution path is:

1. getHostMntNS
2. Parse
3. CreatePolicies
4. GetTraceeRunner
5. main (tracee, tracee-ebpf)

So it is executed before we drop privileges (at tracee.Init()).